app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,7 @@
 
     <%= stylesheet_link_tag "https://www-media.stanford.edu/su-identity/css/su-identity.css" %>
     <%= stylesheet_link_tag "https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700" %>
+    <%= stylesheet_link_tag "https://cdn.jsdelivr.net/npm/simple-datatables@latest/dist/style.css" %>
```

Comment:
Do we want to pin to a specific version so that changes are managed by us or are you confident there will never be breaking changes with `@latest`?

Reply:
Pinning seems appropriate as the most recent tagged version is from June and doesn't change our expected behavior. Done.

Reply:
I see you pinned the version of the JS but not of the CSS

Reply:
I clearly needed more coffee. Done.